### DISCUSSION TITLE
Improved icon search through tags

### DISCUSSION THREAD

**Discussion by lahdekorpi** [directus/directus#23884] [Feature Requests]
### Summary

Import a metadata file containing tags for all icons to improve search.

### Motivation

The current search for icons is terrible. You have to hit the exact file name to find what you are looking for, unlike in the [Material Symbols search](https://fonts.google.com/icons?icon.query=grid) that uses tags and shows a lot more options.

### Detailed Design

These tags are available from: https://fonts.google.com/metadata/icons

### Requirements List

- Better search

### Drawbacks

Slower search

### Alternatives

None.

### Adoption Strategy

Doesn't change existing usage, just improves it.

**Comment by PetrBrabec** [comment]
Yes please!
